shell/server/pre-meteor.js

```diff
@@ -591,12 +630,25 @@ Meteor.startup(() => {
 
         return true;
       } else {
-        const id = matchWildcardHost(req.headers.host);
-        if (id) {
-          return tryProxyUpgrade(id, req, socket, head);
-        } else {
-          return false;
-        }
+        return inMeteor(function () {
+          // TODO(someday): This is slightly sad that some requests will now need a Meteor fiber
```

Comment:
I'm more worried about the DB query.

Note that we could have all DDP WebSockets go to the main host rather than handle them on the standalone host. This will presumably already happen on Oasis since it sets DDP_DEFAULT_CONNECTION_URL to ddp.oasis.sandstorm.io.